pkg/tools/etcd_helper_watch.go

```diff
@@ -250,6 +259,10 @@ func (w *etcdWatcher) translate() {
 			return
 		case res, ok := <-w.etcdIncoming:
 			if ok {
+				if curLen := len(w.etcdIncoming); curLen > 0 {
```

Comment:
@quinton-hoole - this is logged at the 4 level - by default we're using 2, so this won't be logged